word_counter.rb

```diff
@@ -0,0 +1,79 @@
+require 'pragmatic_segmenter'
+# how do you include a gem such as this so that the program will run on a given computer?
+
+def word_counter
+	# Lists words, their frequency, and location based on sentence # from given text doc.
+	# The way the file is read, it could freeze the host if file given is too big.
+	# Will not work with languages other than English
+	#
+	# Started 9/16/2015
+	# Updated 2/14/2016
+	# Author: Christopher Gabriel Manna
+	# find a way to turn this into a class
+	
+	# setting new values
+	h = Hash.new(0)
+	i = 0
+	alph = "a".ord
+	pot_sentences = []
+	words = []
+	
+
+	file_name = ARGV[0]
+	if (File.extname(file_name).to_s == ".txt" && File.exist?(file_name))
+		# https://github.com/diasks2/pragmatic_segmenter
+		# GitHub UN: diasks2
+		txt = File.open(file_name, "r+")
+		ps = PragmaticSegmenter::Segmenter.new(text: txt.read)
+		pot_sentences = ps.segment
+		txt.close
+		# pot_sentences
+		# adjust here so that File Open closes automatically
+	else
+		puts "Something is wrong, please retry."
+		puts "Exitting Program."
+		exit(0)
+	end
+
+
+	# replacing everything that is not a word with nothing
+	pot_sentences.map! { |elmt| elmt.downcase.gsub!(/[^\w\s]/, '') }
+	# rewrite this
+
+	# finding word without gramatical interruption and sorting vals
```

Comment:
This is a super nitpick, but remember it. It's important to detail-oriented programmers: comments about single lines of code should be written as if the subject of the sentence (the line of code) has already been stated. And, they should always start capitalized. So, this comment would become: `Finds word without grammatical interruption and sorting values`...although, I still don't know what that means :stuck_out_tongue_closed_eyes: